tests/misc/tests/misc.js

```diff
@@ -835,4 +835,23 @@ describe("misc", () => {
     const account = await program.account.dataU16.fetch(ifNeededAcc.publicKey);
     assert.ok(account.data, 3);
   });
+
+  it("Can use multidimensional array", async () => {
```

Comment:
I wonder what happens with typescript--though that can go in a separate PR, if there's trouble there.